database/database.js

```diff
@@ -12,53 +12,63 @@ const db = pgp( CONNECTION_STRING )
 const Messages = {
 
   getChatRooms: () => {
-    return db.any(`SELECT * FROM bookstore ORDER BY title ASC`,[])
+    return db.any(`SELECT * FROM chatrooms ORDER BY roomname DESC`,[])
   },
 
-  selectRoom: (id) => {
-    return db.one(`SELECT * FROM bookstore WHERE id=$1`,[id])
+  selectRoom: (roomid) => {
+    return db.one(`SELECT * FROM chatrooms WHERE id=$1`,[roomid])
   },
 
   queryString: (input) => {
     input = `%${input}%`
     return db.any(`
-      SELECT * FROM bookstore
-      WHERE UPPER(title) like UPPER($1)
-      OR UPPER(author) like UPPER($1)
-      OR UPPER(genre) like UPPER($1)`,
+      SELECT * FROM chatrooms
+      WHERE UPPER(roomname) like UPPER($1)
+      `,
       [input])
   },
 
-  leaveChatRoom: id => {
-    return db.none(`DELETE FROM bookstore WHERE id=$1`,[id])
+  leaveChatRoom: activeuserid => {
+    return db.none(`DELETE FROM activeusers WHERE id=$1`,[ activeuserid ])
   },
 
-  createChatRoom: ( { title, author, preview, genre, image_url }) => {
+  createChatRoom: ( roomname ) => {
```

Comment:
very minor change: I would stay consistent with not having () when a func has a single parameter